src/System.Private.ServiceModel/tests/Common/src/TestTypes.cs

```diff
@@ -702,10 +709,16 @@ public Task<Guid> ServicePingCallback(Guid guid)
 
     public Task<Guid> ServicePingFaultCallback(Guid guid)
     {
-        throw new FaultException<FaultDetail>(
-          new FaultDetail("Throwing a Fault Exception from the Callback method."), 
-          new FaultReason("Reason: Testing FaultException returned from Duplex Callback"), 
-          new FaultCode("ServicePingFaultCallback"),
-          "http://tempuri.org/IWcfDuplexTaskReturnCallback/ServicePingFaultCallback");
+        var fault = new FaultException<FaultDetail>(
+            new FaultDetail("Throwing a Fault Exception from the Callback method."),
+            new FaultReason("Reason: Testing FaultException returned from Duplex Callback"),
+            new FaultCode("ServicePingFaultCallback"),
+            "http://tempuri.org/IWcfDuplexTaskReturnCallback/ServicePingFaultCallbackFaultDetailFault");
+        if (_wrapExceptionInTask)
+        {
+            return Task.FromException<Guid>(fault);
```

Comment:
It breaks for me too.  FromException<T> is not in public contract in 4.0.0 of Threading.Tasks but is in the contracdt of 4.0.10.  I see some of the .lock.json files take a dependency on 4.0.0, but I don't see that in the project.json files.  Something in the dependency resolution is bringing in 4.0.0 for this test.